File: AirlineBookingBackend/imports/methods/plane.js
import { Mongo } from 'meteor/mongo';

export const Planes = new Mongo.Collection('planes');

Meteor.methods({
  'plane.addPlane'(planeName, planeNumber, planeType, seatingCapacity) {
      var planeDetails = {
        planeName: planeName,
        planeNumber: planeNumber,
        planeType: planeType,
        seatingCapacity: seatingCapacity
      }

      Planes.insert(planeDetails)
    }
})

File: AirlineBookingBackendPython/app/BookingApp/routes/plane.py
from BookingApp import app
from flask import request
from BookingApp.utils import response_utils as rp
from BookingApp.utils import db_utils as dbu
import json


@app.route('/addPlane', methods=['POST'])
def add_plane():
    data = json.loads(request.data)
    
    if 'planeName' in data:
    	dbu.insert_one({
    		'planeName': data['planeName'],
    		'planeNumber': data['planeNumber'],
            'planeType': data['planeType'],
            'seatingCapacity': data['seatingCapacity']
    	}, 'planes')


    return rp.create_response({})

@app.route('/getPlanes')
def get_planes():
    data = dbu.find({}, 'planes')

    return rp.create_response(data)